src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -53,6 +53,8 @@ internal AssemblyScanner(Assembly assemblyToScan)
 
     internal string CoreAssemblyName { get; set; } = NServiceBusCoreAssemblyName;
```

Comment:
I kept this around because someone might use this via reflection. To keep things consistent, I added another internal property for now.